tests/components/mpd/conftest.py

```diff
@@ -31,7 +31,7 @@ def mock_setup_entry() -> Generator[AsyncMock, None, None]:
 
 
 @pytest.fixture
-def mock_mpd_client() -> Generator[AsyncMock, None, None]:
+def mock_mpd_client() -> Generator[AsyncMock]:
```

Comment:
```suggestion
def mock_mpd_client() -> Generator[Mock]:
```

For this one, it would be really good to use `Generator[MockOf[MPDClient]]` but it's not available yet.